Microsoft.Toolkit.Uwp.SampleApp/SamplePages/BluetoothLEHelper/BluetoothLEHelperCode.bind

```diff
@@ -0,0 +1,12 @@
+﻿// Get a local copy of the context for easier reading
+BluetoothLEHelper bluetoothLEHelper = BluetoothLEHelper.Context;
+
+// Start the Enumeration
+bluetoothLEHelper.StartEnumeration();
+
+// Connect to a device if your choice
+ObservableBluetoothLEDevice device = 
+bluetoothLEHelper.BluetoothLeDevices[<Device you choose>].Connect()
```

Comment:
How do you want me to do this? bluetoothLEHelper.BluetoothLeDevices is a collection of all LE devices in the area that are connectable. The device that the user wants to connect to has to be choosen somehow, but I thought the UI and such were out of scope for these code sample files.